l10n_es_intrastat_report/__manifest__.py

```diff
@@ -24,6 +24,7 @@
         "views/l10n_es_intrastat_product.xml",
         "wizards/l10n_es_intrastat_code_import.xml",
     ],
+    "external_dependencies": {"python": ["xlrd"]},
```

Comment:
You don't need to add this one, as it's an Odoo base library:

https://github.com/odoo/odoo/blob/3cbb5ad003aa58ed85df182b136ed38d9a0e4ede/requirements.txt#L46-L47

Reply:
Vaya, de repente pre-commit empezó a darme mensajes que no se reflejaban en runboat. Debe estar relacionado con los cambios que hice en la rama 16.0 de [oca/connector-prestashop](https://github.com/oca/connector-prestashop/)